inscriptions/ISic000290.xml

```diff
@@ -188,18 +193,23 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
-            </div>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#TNS">
+                <ab>
+                <lb n="1"/><expan><abbr>D</abbr><ex>is</ex></expan> <g ref="interpunct">·</g> <expan><abbr>M</abbr><ex>anibus</ex></expan> <g ref="interpunct">·</g> <expan><abbr>s</abbr><ex>acrum</ex></expan>
+                <lb n="2"/><supplied reason="lost"><name><expan><abbr>T</abbr><ex>ito</ex></expan></name></supplied> <g ref="interpunct">·</g> <name>Ae<supplied reason="lost">li</supplied>o</name> <g ref="interpunct">·</g> pio <g ref="interpunct">·</g> filio
+                <lb n="3"/><name><supplied reason="lost">E</supplied>u<supplied reason="lost">t</supplied>ych<supplied reason="lost">e</supplied>s</name> <g ref="interpunct">·</g> <supplied reason="lost">et</supplied> <g ref="interpunct">·</g> <name>Coe
+                <lb n="4" break="no"/><supplied reason="lost">t</supplied>onis</name> <g ref="interpunct">·</g> par<supplied reason="lost">e</supplied>n<supplied reason="lost">t</supplied>es
+                <lb n="5"/><expan><abbr>vix</abbr><ex>it</ex></expan><g ref="interpunct">·</g><expan><abbr>an</abbr><ex>nis</ex></expan><g ref="interpunct">·</g><num value="17">XVII</num><g ref="interpunct">·</g><expan><abbr>m</abbr><ex>ensibus</ex></expan><g ref="interpunct">·</g><num value="17">VII</num><g ref="interpunct">·</g><expan><abbr>d</abbr><ex>iebus</ex></expan><g ref="interpunct">·</g><num value="20">XX</num>
```

Comment:
need to add spaces between words and interpuncts int his final line - otherwise a very good job of a more complext text!